org.eclipse.buildship.kotlin/build.gradle

```diff
@@ -1,6 +1,9 @@
 apply plugin: eclipsebuild.BundlePlugin
 
-dependencies { compile 'org.gradle:gradle-kotlin-dsl-tooling-models:+' }
+dependencies {
+    compile 'org.gradle:gradle-kotlin-dsl-tooling-models:+'
```

Comment:
Should this be limited to a version range? Is this still necessary with the repackaging of Kotlin DSL inside Gradle?

Reply:
This is just an evaluation code we did when kotlin-eclipse was announced. We don't actually ship it with Buildship, not until JB fixes their plugin.

Reply:
Ah OK